app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinFragment.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.android.app.settings.profile
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.FragmentComponentImpl
+import org.oppia.android.app.fragment.InjectableFragment
+import org.oppia.android.domain.profile.ProfileManagementController
+import javax.inject.Inject
+
+/** Fragment that resets the profile pin of the user
+ */
+class ProfileResetPinFragment : InjectableFragment() {
```

Comment:
Nit: ** Fragment that resets the profile pin of the user. */

Reply:
done